README.md

```diff
@@ -12,14 +12,32 @@ The Sparkmagic project includes a set of magics for interactively running Spark
 ## Features
 
 * Run Spark code in multiple languages against any remote Spark cluster through Livy
-
+* Automatic SparkContext (`sc`) and HiveContext (`sqlContext`) creation
+* Ability to execute SparkSQL queries
```

Comment:
This isn't really a feature, since you can run SparkSQL queries from anywhere (including a plain Python2 notebook).  You can say something like "easily execute SparkSQL queries with the `%%sql` magic"